superset/explore/permalink/commands/create.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+import logging
+from typing import Any, Dict, Optional
+
+from flask_appbuilder.security.sqla.models import User
+from flask_babel import gettext as _
+from sqlalchemy.exc import SQLAlchemyError
+
+from superset.explore.permalink.commands.base import BaseExplorePermalinkCommand
+from superset.explore.permalink.exceptions import ExplorePermalinkCreateFailedError
+from superset.explore.utils import check_access
+from superset.key_value.commands.create import CreateKeyValueCommand
+from superset.key_value.types import KeyType
+
+logger = logging.getLogger(__name__)
+
+
+class CreateExplorePermalinkCommand(BaseExplorePermalinkCommand):
+    def __init__(  # pylint: disable=too-many-arguments
```

Comment:
Should we follow the same pattern of `temporary_cache/commands/CommandParameters`?